### DISCUSSION TITLE
LVGL big fonts.

### DISCUSSION THREAD

**Discussion by utya** [arendst/Tasmota#18808] [Q&A]
Hi ,
i want use utf-8 fonts. please explain me, how to use?
In some case i want big special symbol, how can i do it?

**Comment by s-hadinger** [comment]
I would recommend to use integrated FreeType support. Please note it requires PSRAM

https://tasmota.github.io/docs/LVGL/#freetype-fonts-support

**Reply by utya** [reply]
psram, don't have.

**Reply by s-hadinger** [reply]
So there is no way you can have big fonts because memory is too limited

**Reply by utya** [reply]
Yes, I have a simple task to display, OK or ERROR. As I understand it, I won’t be able to use the text without memory, well, I thought maybe with special characters?

**Reply by s-hadinger** [reply]
Without PSRAM you can only use already available fonts from Flash. From the documentation:

Fonts, currently Montserrat fonts are embedded at sizes 10, 14 (default), 20 and 28 (compressed - smaller and slower)

**Reply by utya** [reply]
can use this solution?

#- start LVGL and init environment -#
lv.start()

hres = lv.get_hor_res()       # should be 320
vres = lv.get_ver_res()       # should be 240

scr = lv.scr_act()            # default screan object
f20 = lv.montserrat_font(28)  # load embedded Montserrat 20

#- Background with a gradient from black #000000 (bottom) to dark blue #0000A0 (top) -#
# scr.set_style_bg_color(lv.color(0x0000A0), lv.PART_MAIN | lv.STATE_DEFAULT)
stat_line = lv.label(scr)
stat_line.set_long_mode(lv.LV_LABEL_LONG_BREAK) <--------  Problem in this Line
stat_line.set_text(lv.SYMBOL_OK);

**Reply by s-hadinger** [reply]
Also read here about special symbols already included

https://docs.lvgl.io/8.3/overview/font.html

**Reply by s-hadinger** [reply]
Yes. Symbols are included
